### PR TITLE
Day 10 - Slate Python

### PR DESCRIPTION
**Pull request by 5late**

### CODE REVIEW

File: 2021/9/solutions/python/slate/example.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+2199943210
```
Comment (stelar7) [inline 2021/9/solutions/python/slate/example.txt:1]: delete

File: 2021/10/solutions/python/slate/10d.txt [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{[<<([[{{[<[[[<>()]{[]<>}]]<[[{}()>{[]{}}]>>][((<{{}[]}{[][]}>({{}{}}{{}[]})))([{({}[])}(<
```
Comment (stelar7) [inline 2021/10/solutions/python/slate/10d.txt:1]: delete

File: 2021/10/solutions/python/slate/d10.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import sys
```
Comment (stelar7) [inline 2021/10/solutions/python/slate/d10.py:1]: nice day 10 lul

Comment (5late) [reply 2021/10/solutions/python/slate/d10.py:1]: ``git add .``

**Comment by 5late** [comment]
Day 9 is gone. RIP

**Pull request merged**